docs/how_to/debug_beaker_tests.md

```diff
@@ -1,19 +1,23 @@
-## What's Pry?
+# Debug beaker tests
 
-[Pry](http://pryrepl.org/) is a powerful Ruby editing and debugging tool.  Beaker uses Pry runtime invocation to create a developer console.
+beaker includes [pry-byebug](https://github.com/deivid-rodriguez/pry-byebug), a gem that combines two powerful related tools: pry and byebug
+
+## Pry
 
-## Set it up!
+### What's Pry?
+
+[Pry](http://pryrepl.org/) is a powerful Ruby editing and debugging tool.  Beaker uses Pry runtime invocation to create a developer console.
 
-Pry is included in the Beaker 1.13+ development gems and is available by default.  
+### Set it up!
 
-Pre-1.13.0 add Pry with `gem install pry`, then add Pry to individual tests by adding `require 'pry'` as the first line in the Ruby test file.
+Add Pry to individual tests by adding `require 'pry'` as the first line in the Ruby test file.
```

Comment:
Have you thought about requiring pry from the primary [beaker.rb](https://github.com/puppetlabs/beaker/blob/master/lib/beaker.rb) file? It seems like requiring pry per individual test is a bit of a hassle.